todo_app/templates/index.html

```diff
@@ -7,32 +7,11 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
-  <div class="row">
-    <div class="col-md-12">
-      <h2>Add Item</h2>
-        <form action="/additem" method="post" class="input-form">
-          <div class="input-form">
-            <label for="title">Enter your to-do item: </label>
-            <input type="text" name="title1" id="name" required />
-          </div>
-          <div class="input form">
-            <input type="submit" value="Add Item" />
-          </div>
-        </form>
-    </div>
-  </div>
-
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-          <ul>
-            <li>
-              {% for i in list1 %}
```

Comment:
I'm a fan of really descriptive names, so e.g. rather than `list1`, `todo_items`, or `list_of_todo_items` etc., and rather than `i` `todo_item`, `todo_title` etc.?